app/assets/stylesheets/hybox.scss

```diff
@@ -1,26 +1,141 @@
-$brand-primary: #629dd1 !default;
-$jumbotron-heading-font-size: 2.5em;
-$jumbotron-heading-color: #fff;
-$jumbotron-color: #d8d8d8;
 
-.jumbotron {
-  background-image: url("http://hydrainabox.projecthydra.org/images/dark_tint.png"), url("http://hydrainabox.projecthydra.org/images/fractal.jpg");
+#masthead,
+#footer {
+  background-color: #3c3c3c;
+  border: 0;
+
+  a,
+  p {
+    color: #ccc;
+    font-weight: 500;
+  }
+}
+
+#masthead {
+  #logo {
+    margin-left: 10px;
+  }
+
+  .navbar-right {
+    margin-right: 10px;
+  }
 
-  font-family: "Roboto", Helvetica, sans-serif;
+  .glyphicon {
+    display: none;
+  }
+}
+
+.jumbotron {
+  background-color: #eee;
+  background: url('white-cloud-background.jpg') repeat-x;
+  border: 0;
+  font-family: "Lato", Helvetica, sans-serif;
+  margin-bottom: 0;
   text-align: center;
 
   h1 {
-    font-weight: 300;
-    letter-spacing: 0.125em;
+    color: #30373b;
+    font-weight: 700;
+    letter-spacing: 0.075em;
     text-transform: uppercase;
   }
+
+  p {
+    color: #444;
+    font-size: 1.6em;
+    margin-bottom: 20px;
+  }
+
+  .btn-sign-up {
```

Comment:
Can we use `button-variant` (from https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/mixins/_buttons.scss#L6) instead of using the compiled styles directly?